katalog/node-exporter/deploy.yml

```diff
@@ -41,11 +44,37 @@ spec:
           mountPropagation: HostToContainer
           name: root
           readOnly: true
+      - args:
```

Comment:
Do we need this sidecar?

Reply:
I think it makes sense, afterall node_exporter is running with `hostNetwork: true` so anyone in the same subnet can `curl` its metrics discolsing informations about the system.